stable/mongodb-replicaset/README.md

```diff
@@ -127,13 +132,16 @@ configmap:
     port: 27017
     ssl:
       mode: requireSSL
-      CAFile: /ca/tls.crt
+      CAFile: /data/configdb/tls.crt
       PEMKeyFile: /work-dir/mongo.pem
+      # Set to false to require mutual TLS encryption
+      allowConnectionsWithoutCertificates: true
   replication:
     replSetName: rs0
   security:
     authorization: enabled
-    clusterAuthMode: x509
```

Comment:
is this a breaking change?  Should we be bumping a major version ?

Reply:
No it should not break backwards compatibility - it simply enables auth to be one-way TLS versus mutual TLS.  People can still specify x509 mode if they want.

Reply:
:+1: